tests/src/common.rs

```diff
@@ -0,0 +1,61 @@
+use serde:: {Serialize, Deserialize};
+use anyhow::Result;
+use rand::{distributions::Alphanumeric, Rng};
+
+pub async fn send_get(
+    url: &str
+) -> Result<reqwest::Response, reqwest::Error> {
```

Comment:
Types in reqwest e.g. reqwest::Response, reqwest::Error, reqwest::Client can `use reqwest::{Response, Error, Client};`

Reply:
Beginner error :-)